access/templates/access/graded_form.html

```diff
@@ -1,32 +1,39 @@
 {% load i18n %}
-<form method="post" action="{{ post_url }}"{% if result.form.multipart %} enctype="multipart/form-data"{% endif %}>
+{% load access %}
+<form method="post" action="{{ post_url }}"{% if result.form.multipart %} enctype="multipart/form-data"{% endif %} class="aplus-quiz1">
 	{% for field in result.form %}
+
 	{% if field.field.open_set %}
-	<fieldset id="{{ field.field.open_set }}"{% if result.accepted and field.field.group_errors %} class="alert {% if field.field.open_set in result.error_groups %}alert-danger{% else %}alert-success{% endif %}"{% endif %}>
+	<!-- Beginning of all questions -->
+		<fieldset id="{{ field.field.open_set }}"{% if result.accepted and field.field.group_errors %} class="alert {% if field.field.open_set in result.error_groups %}alert-danger{% else %}alert-success{% endif %}"{% endif %}>
```

Comment:
Do we want to use default bootstrap alerts for this?

Reply:
I didn't really touch that line at this point, but I could split it, for sure. 

And I can take a closer look at that alert-class here: https://github.com/apluslms/mooc-grader/pull/78. Remove it or change it to the one I proposing there, or to something else. Looks like appropriate place.

Reply:
I didn't really touch that here, but I think you are right, it could have some work on here as well.
 
What it actually currently does (in the current production version as well) is this:

![image](https://user-images.githubusercontent.com/43036333/89494944-e6da1a00-d7bf-11ea-96e7-dc0a902a50ba.png)

![image](https://user-images.githubusercontent.com/43036333/89494951-eb9ece00-d7bf-11ea-90d2-de5423001293.png)

Based on the documentation, it is not what is expected:
https://github.com/apluslms/mooc-grader/blob/master/courses/README.md#accesstypesstdsynccreateform

I removed the alert from it, it was unnecessary to wrap the whole fieldset in an alert if there are some hidden failed fields.